packages/xo-web/src/xo-app/hub/recipes/recipe-form.js

```diff
@@ -100,6 +108,78 @@ export default decorate([
           value={value.nbNodes}
         />
       </FormGrid.Row>
+      <FormGrid.Row>
+        <label>
+          {_('recipeStaticIpAddresses')}
+          <input
+            className='form-control'
+            name='staticIpAddress'
+            onChange={effects.toggleStaticIpAddress}
+            type='checkbox'
+            value={value.staticIpAddress}
+          />
```

Comment:
We use the same style for checkbox in XO5  `checkbox label` in the same row

Reply:
I'm sorry Rajaa but I don't understand what is expected.
Is there any difference between what I did and that?: https://github.com/vatesfr/xen-orchestra/blob/d7794518a2a21dac7ecbc0a7146c11928799dd7a/packages/xo-web/src/xo-app/new/sr/index.js#L743-L751
Maybe you can show me an example.